Sources/Hub/Hub.swift

```diff
@@ -112,9 +112,15 @@ public class LanguageModelConfigurationFromHub {
 
     private var configPromise: Task<Configurations, Error>? = nil
 
-    public init(modelName: String) {
+    public init(
+        modelName: String,
+        hubApi: HubApi = HubApi()
+    ) {
         self.configPromise = Task.init {
-            return try await self.loadConfig(modelName: modelName)
+            return try await self.loadConfig(
+                modelName: modelName,
+                hubApi: hubApi
+            )
```

Comment:
(Personally not a fan of multi-lines, especially if they're short, but let's not worry about that for the moment)